plugin-build/src/main/kotlin/io/sentry/android/gradle/util/SentryPluginUtils.kt

```diff
@@ -49,4 +52,10 @@ internal object SentryPluginUtils {
         }
         return variant.buildType.isMinifyEnabled
     }
+
+    fun getAndDelete(property: Provider<RegularFile>): File {
```

Comment:
`l`: I'd make the method name more descriptable, how about `getAndDeleteFile()`?